web/docs/guides/platform/logs.mdx

```diff
@@ -63,18 +51,13 @@ WHERE h.x_real_ip IS NOT NULL
 ![SELECT query example](/img/guides/platform/logs/select-query.png)
 
 
-You may notice that the table name reference, `edge_logs`, is required in order to formulate the SELECT query. Likewise, when querying the database logs, you would need to provide a table name reference as well.
-
-The table references are as follows:
-
-- API Edge - `edge_logs`
-- Database - `postgres_logs`
+The log source, `edge_logs`, is required in order to formulate the SELECT query. Likewise, when querying the database logs, you would need to provide a table name reference as well. The list of supported product sources can be found under the **Sources** dropdown.
 
-Note that cross-table joining is currently not supported.
+Cross-source joining is currently supported.
```

Comment:
seems like an unnecessary statement

Or, if you feel it is necessary: `Cross-source joining is supported.`. No need for `currently`